### ISSUE TITLE
[Bug]: Reliquary Mob Drops

### ISSUE THREAD

**Issue by Kithio** [FTBTeam/FTB-Modpack-Issues#1410]
### Modpack

FTB Inferno

### Modpack version

1.0.2

### Describe the bug

Reliquary drops like Slime Pearl; Catalyzing Gland, Nebulous Heart - Disabled in config? and no mentioned way of how to get them, even tho the Reliquary crafting recipies are still all enabled
- I assumed by having things like Tome of Alkahestry shown; It'd be an obtainable item and given the various rare mob drops are still shown and some are even obtainable through other means, but it doesn't seem all of them would be obatainable

Suggestion : Either enable the drops or add a specified way to get the various parts needed for the recipes; or Hide/Remove them  using KubeJS to avoid confusion in future 

### Steps to reproduce

- No necessary steps, Config is disabled but some items are gained through other means, most recipes are unusable due to missing pieces.

### Expected behaviour

Reliquary Mob Drops Obtainable

### Screenshots

![image](https://user-images.githubusercontent.com/9414786/194771712-ecaa446c-5e55-4513-9974-899951a02645.png)
![image](https://user-images.githubusercontent.com/9414786/194771721-0def3170-7d9a-4867-b08b-01007e285d27.png)
![image](https://user-images.githubusercontent.com/9414786/194771741-c6935f04-e013-4b1a-a55b-238a97b46598.png)
![image](https://user-images.githubusercontent.com/9414786/194771767-95055b1f-7d97-4c26-aa0a-3ea474a9b9a5.png)
![image](https://user-images.githubusercontent.com/9414786/194771802-b8ccd67c-ae90-4363-95e0-adc9f7c66e84.png)

### Additional information

As shown, a couple of the items can be gained through alternate means, but items like Slime Pearl has no listed way of getting them, and also does not drop from mobs as Reliquary Mob Drops are disabled while Magma Core can be gained from Fishing and Chelicerae is gained from a quest as a reward

- I've looked through all available quests and active scripts and see nothing that would reward the other missing items to do a fair few of the recipes.

**Comment by Raidobw2** [comment]
Thanks for reporting

**Comment by Raidobw2** [comment]
There has been a comment on the linked issue ^ about an item as well:
> Also maybe increase the charm fragment drop chance, they seem REALLY rare.

**Comment by Saereth** [comment]
yeah there are still a few more items that need to find a home in loot/trading tables probably wont be in until 1.0.4 however
